### PR TITLE
added: navbar component

### PR DESCRIPTION
**Pull request by rkapoor10**
Added the following features  - 
- Basic Navbar Structure
navbar component

- Navbar component - with Responsiveness
navbar

### CODE REVIEW

File: src/components/shared/navbar/NavIcons.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { FaShoppingCart } from "react-icons/fa";
+import { FaSearch } from "react-icons/fa";
+import { BsHeart } from "react-icons/bs";
+import NavIconsStyles from "../NavIconsStyles";
+
+const NavIcons = () => {
```
Comment (JantuDeb) [inline src/components/shared/navbar/NavIcons.jsx:8]: NavLinks would be a good component name I guess.

**Review by JantuDeb — APPROVED** [review]
It's good that you are making smaller components. I would you suggest you not to create huge list of components. Create components whenever it is necessary

File: src/components/shared/BrandTitle.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
```
Comment (Raghav888) [inline src/components/shared/BrandTitle.jsx:1]: You hvae created small small components. that's great

**Review by Raghav888 — APPROVED** [review]
Structure is good

**Pull request merged**